pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -487,6 +445,19 @@ def _has_valid_annotations(
     return False
 
 
+def _get_module_platform(module_name: str) -> str | None:
+    """Called when a Module node is visited."""
+    if not (module_match := _MODULE_REGEX.match(module_name)):
+        # Ensure `homeassistant.components.<component>`
+        # Or `homeassistant.components.<component>.<platform>`
+        return None
+
+    platform = module_match.groups()[0]
+    if platform is None:
+        return "__init__"
+    return platform[1:]
```

Comment:
```suggestion
    return platform.lstrip(".") if platform else "__init__"
```
To remove the `.`, I would recommend either `lstrip(".")` or `removeprefix`.

The shortened if expression looks nice, but that is personal preference.

Reply:
done